src/main/groovy/com/modelsolv/reprezen/soapui/RepreZenExporter.groovy

```diff
@@ -82,18 +85,21 @@ class RepreZenExporter {
 		resourceAPI.name = normalize(name)
 		resourceAPI.baseURI = baseUri
 		zenModel.resourceAPIs.add(resourceAPI)
-		exportRestService( service, resourceAPI)
+		DataModel dataModel = datatypesFactory.createDataModel()
+		dataModel.setName(normalize(name)+"DataModel")
+		zenModel.getDataModels().add(dataModel)
+		exportRestService( service, resourceAPI, dataModel)
 		return zenModel
 	}
 
-	private def exportRestService(RestService restService, ResourceAPI resourceAPI) {
+	private def exportRestService(RestService restService, ResourceAPI resourceAPI, DataModel dataModel) {
 		restService.resourceList.each {
-			def ResourceDefinition res = createResourceDefinition( it, resourceAPI )
+			def ServiceDataResource res = createResourceDefinition( it, resourceAPI, dataModel)
 			resourceAPI.ownedResourceDefinitions.add(res)
 		}
 	}
 
-	private def ResourceDefinition createResourceDefinition( RestResource resource, ResourceAPI resourceAPI ) {
+	private def ServiceDataResource createResourceDefinition( RestResource resource, ResourceAPI resourceAPI, DataModel dataModel ) {
 		ServiceDataResource result = restapiFactory.createObjectResource()
```

Comment:
Again, I was referring to the post-processing logic in Swagger import, which infers resource-level data structure bindings from the methods and their message payloads.  Clearly this is not possible now because we don't have message payload models from Ready API.